src/main.js

```diff
@@ -0,0 +1,10 @@
+import { createApp } from 'vue'
+import App from './App.vue'
+import router from './router'
+import store from './store'
+
+import "bootstrap";
+import "bootstrap/dist/css/bootstrap.min.css";
+
```

Comment:
You need to include the CSS file of vue-multiselect in your project.

`import "vue-multiselect/dist/vue-multiselect.css"`